lib/common/utils.ts

```diff
@@ -11,50 +11,51 @@
  * @suppress {undefinedVars,globalThis,missingRequire}
  */
 
+// issue #989, to reduce bundle size, use short name
+export const a = Object.getOwnPropertyDescriptor;
+export const b = Object.defineProperty;
+export const c = Object.defineProperties;
+export const d = Object.getPrototypeOf;
+export const e = Object.create;
+export const f = Array.prototype.slice;
+export const g = 'addEventListener';
+export const h = 'removeEventListener';
+export const i = (Zone as any).s(g);
+export const j = (Zone as any).s(h);
+export const k = 'true';
+export const l = 'false';
+export const m = '__zone_symbol__';
+export const n = 'function';
```

Comment:
i think `typeof x == 'string` is faster than `typeof x == STRING_CONST`. That is because in one case the jitter can create the check code which directly checks type to be string, in tho other case it has to generate a more complex code which could be actual type and than compare it to the constant. I would revert this.

Reply:
Got it, I have reverted all `typeof x ===` parts.